### ISSUE TITLE
fabmanager-logo.png problem manifest.json

### ISSUE THREAD

**Issue by shak17** [sleede/fab-manager#335]
Hello, fresh install Ubuntu 18.04, having problem on mail module:
Sidekick report:
***************
OverwrittenDeviseMailer#reset_password_instructions |
 {"_aj_globalid"=>"gid://fablab/User/1"}, "fXR1AtUfBayELp-WyoQU", {"_aj_symbol_keys"=>[]} 

 ActionView::Template::Error: Webpacker can't find  media/images/fabmanager-logo.png in  /usr/src/app/public/packs/manifest.json. Possible causes: 1. You forgot to install node packages (try `yarn install...
**************

on file /apps/fabmanager/public/manifest.json is well declared, also file is present  

fabmanager-logo-0dc020701a1d12b85b03.png

  "static/fabmanager-logo.png": "/packs/static/fabmanager-logo-0dc020701a1d12b85b03.png",

tested in 20.04, same problem. How to solve it ?

**Comment by sylvainbx** [comment]
Hi @shak17 ,
What's the output of the following commands :
```bash
ls /apps/fabmanager/public/packs/static

grep packs /apps/fabmanager/docker-compose.yml
```

**Comment by shak17** [comment]
```
root@localhost:~# ls /apps/fabmanager/public/packs/static
arrow-left-6cfe3d9e2c18a348cde1.png
bootstrap-sass
@claviska
fabmanager-logo-0dc020701a1d12b85b03.png
@fortawesome
mastercard-a0dda355ba619ab1f12e.png
no_avatar-5997afd4951fd3ce4e62.png
payzen-secure-f2f2c2fb967b78ca99c9.png
powered_by_stripe-a43f123c8b2030f5781f.png
social
summernote
visa-e31b5cb84a808cb0dfd0.png

root@localhost:~# grep packs /apps/fabmanager/docker-compose.yml
      - ${PWD}/public/packs:/usr/src/app/public/packs
```

**Comment by sylvainbx** [comment]
Everything seems ok to my eyes.
Have you tried to restart with the following command?
```
docker-compose down && docker-compose up -d
```

**Comment by shak17** [comment]
Hello, yes, I tried several times and no solution to this issue. I cannot go further because is not possible to communicate externally with users.

for information, I have three warnings during install :

****************************************************
WARNING in asset size limit: The following asset(s) exceed the recommended size li                                                                                      mit (244 KiB).
This can impact web performance.
Assets:
  static/@fortawesome/fontawesome-free/webfonts/fa-brands-400-63b566233f163d538cce                                                                                      .svg (713 KiB)
  static/@fortawesome/fontawesome-free/webfonts/fa-solid-900-3e2bc2ffd29f01066d23.                                                                                      svg (893 KiB)
  css/application-3b47bbce.css (442 KiB)
  js/application-09079bc4bf92aa0d7af3.js (1.13 MiB)
  js/868-b6d74d30301045b0052c.js (3.68 MiB)
  static/@fortawesome/fontawesome-free/webfonts/fa-solid-900-3e2bc2ffd29f01066d23.                                                                                      svg.gz (249 KiB)
  js/application-09079bc4bf92aa0d7af3.js.map.gz (431 KiB)
  js/868-b6d74d30301045b0052c.js.gz (971 KiB)
  js/868-b6d74d30301045b0052c.js.map.gz (3.6 MiB)
  js/application-09079bc4bf92aa0d7af3.js.map.br (322 KiB)
  js/868-b6d74d30301045b0052c.js.br (765 KiB)
  js/868-b6d74d30301045b0052c.js.map.br (2.77 MiB)

WARNING in entrypoint size limit: The following entrypoint(s) combined asset size                                                                                       exceeds the recommended limit (244 KiB). This can impact web performance.
Entrypoints:
  application (5.24 MiB)
      js/runtime-d66b648ac2302ca23b08.js
      js/868-b6d74d30301045b0052c.js
      css/application-3b47bbce.css
      js/application-09079bc4bf92aa0d7af3.js

WARNING in webpack performance recommendations:
You can limit the size of your bundles by using import() or require.ensure to lazy                                                                                       load some parts of your application.
For more info visit https://webpack.js.org/guides/code-splitting/

webpack 5.70.0 compiled with 3 warnings in 272108 ms

**********************************************************

Do you think the problem is related to this ?

**Comment by gnepud** [comment]
Hello @shak17

I saw this problem. You can try delete all assets and reboot your server.
```
cd /apps/fabmanager/
docker-compose down
rm -rf public/packs
reboot
cd /apps/fabmanager/
docker-compose up -d
docker-compose run --rm REPLACE_YOUR_FABMANAGER_NAME bundle exec rake assets:precompile
```

**Comment by shak17** [comment]
problem still there : 

*********************
WARNING in asset size limit: The following asset(s) exceed the recommended size limit (244 KiB).
This can impact web performance.
Assets:
  static/@fortawesome/fontawesome-free/webfonts/fa-brands-400-63b566233f163d538cce.svg (713 KiB)
  static/@fortawesome/fontawesome-free/webfonts/fa-solid-900-3e2bc2ffd29f01066d23.svg (893 KiB)
  css/application-3b47bbce.css (442 KiB)
  js/application-09079bc4bf92aa0d7af3.js (1.13 MiB)
  js/868-b6d74d30301045b0052c.js (3.68 MiB)
  static/@fortawesome/fontawesome-free/webfonts/fa-solid-900-3e2bc2ffd29f01066d23.svg.gz (249 KiB)
  js/application-09079bc4bf92aa0d7af3.js.map.gz (431 KiB)
  js/868-b6d74d30301045b0052c.js.gz (971 KiB)
  js/868-b6d74d30301045b0052c.js.map.gz (3.6 MiB)
  js/application-09079bc4bf92aa0d7af3.js.map.br (322 KiB)
  js/868-b6d74d30301045b0052c.js.br (765 KiB)
  js/868-b6d74d30301045b0052c.js.map.br (2.77 MiB)

WARNING in entrypoint size limit: The following entrypoint(s) combined asset size exceeds the recommended limit (244 KiB). This can impac                                                                                                    t web performance.
Entrypoints:
  application (5.24 MiB)
      js/runtime-d66b648ac2302ca23b08.js
      js/868-b6d74d30301045b0052c.js
      css/application-3b47bbce.css
      js/application-09079bc4bf92aa0d7af3.js

WARNING in webpack performance recommendations:
You can limit the size of your bundles by using import() or require.ensure to lazy load some parts of your application.
For more info visit https://webpack.js.org/guides/code-splitting/

webpack 5.70.0 compiled with 3 warnings in 272936 ms

******************************

![Screenshot 2022-04-07 at 17-19-58 Sidekiq](https://user-images.githubusercontent.com/1121673/162234143-558ee577-6210-4dfd-b3ec-5cdfebfdac1d.png)

**Comment by gnepud** [comment]
Could you try update your docker to v20 ? Make sure re-complie assets and reboot server after update.

**Comment by shak17** [comment]
Docker is already in version 20 :  20.10.14

I did recompile all as was described here https://github.com/sleede/fab-manager/issues/335#issuecomment-1091817512

![image](https://user-images.githubusercontent.com/1121673/162239168-707f9fa5-aa66-498f-aad6-cd18716e5bf6.png)

**Comment by shak17** [comment]
J'ai essayé chez OVH même problème. J'ai essayé Ubuntu 18.04, 20.04, et toujours le même problème. Une piste ? Depuis 3 semaine j'essaie de mettre en place fabmanager sans pouvoir aboutir.

**Comment by sylvainbx** [comment]
Est-ce que tu peux essayer de lancer un prompt dans le container et de regarder si il arrive bien à accéder aux fichiers ?

```bash
docker-compose up -d
docker-compose exec fabmanager bash
ls -la /usr/src/app/public/packs/
```

**Comment by nflorentin** [comment]
Salut @shak17 , 

Nous avons fait un fix il y a 3 semaines au sujet de ce problème. 

Peux-tu essayer avec la dernière version de fab-manager ?

Aussi, tu peux ignorer les warnings de compilation qui ne sont probablement pas liés à ton soucis.

**Comment by sylvainbx** [comment]
Should be fixed in v5.3.11. Closing this issue due to inactivity. Feel free to reopen if you encounter the issue again.
